### PR TITLE
Feature/simple angular l10n

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/5696

### CODE REVIEW

**Comment by myabc** [comment]
@tilt can we close #1082?

File: app/views/layouts/base.html.erb [hunk 1]
```diff
@@ -43,6 +43,7 @@ See doc/COPYRIGHT.rdoc for more details.
 <% end %>
 
 <%= javascript_include_tag 'application' %>
+<%= javascript_include_tag 'angular-i18n/angular-locale_de-de' if I18n.locale == :de %>
```
Comment (myabc) [inline app/views/layouts/base.html.erb:46]: @tilt we'll need to add this to `config.assets.precompile`.

Comment (tilt) [inline app/views/layouts/base.html.erb:46]: of course - you mentioned that, sorry

Comment (myabc) [inline app/views/layouts/base.html.erb:46]: also, do we need to require `angular-i18n/angular-locale_en.js` somewhere?

Comment (tilt) [inline app/views/layouts/base.html.erb:46]: not as far as I knew, it's just for the built-in AngularJS l10n filter methods, so at the very front end - or did I misunderstand your question?

**Pull request merged**